### DISCUSSION TITLE
Dashy 2.0.0

### DISCUSSION THREAD

**Discussion by lissy93** [Lissy93/dashy#459] [Releases]
Version 2.0.0 of Dashy is now released! The main changes are the introduction of widgets, as well as usability + stability improvements.

Thanks very much to everyone who contributed to this release, including: @XertDev, @b1thunt3r, @icy-comet, @KierenConnell, @BOZG, @leocov-dev, @walkxcode, @olearycrew, @Compunctus, @EVOTk, @stuu3k, @jnach, @DylanBeMe

## ✨ New Features
- Widgets! Support for widgets for displaying dynamic content, as well as 50+ pre-built widgets
- Native SSL support, enabling self-signed certificated to be passed directly to the Docker container

## ⚡️ Improvements
- Adds several new themes
- Adds option to change font from UI
- Automated multi-arch Docker builds
- Performance improvements to the Node server
- Switches default favicon API to allesedv, and adds support for iconhorse
- Adds icon fallback support

## 🐛 Bug Fixes
- Fixes overlapping edit menu (#311)
- Fixes status check dot in edit mode (#301)
- Fixes item text flicker on hover (#312)
- Fix unequal button height in French (#311)
- Makes sorting case-insensitive (#352)
- Fixes tile move on hover bug (#366)
- Fixes save item without title bug (#377) 
- Fixes formatting in clock + weather widgets (#402)
- Fixes unable to edit item bug (#415)
- Fixes unable to add new app bug (#390)
- Fixes nav links still visible after removing (#389)
- Fixes clock date update delay at midnight (#402)
- Fixes empty section visible in search (#447)
- Fixes numbers being removed from tags (#430)

<sup>For more info, view the [Changelog](https://github.com/Lissy93/dashy/blob/master/.github/CHANGELOG.md) or this release's [commit history](https://github.com/Lissy93/dashy/compare/1.9.0...2.0.0).</sup>

For a list of upcoming tasks, checkout the [Current Roadmap](https://github.com/Lissy93/dashy/discussions/405)

<hr /><em>This discussion was created from the release <a href='https://github.com/Lissy93/dashy/releases/tag/2.0.0'>Dashy 2.0.0</a>.</em>

**Comment by lammersbjorn** [comment]
💚
